lib/rules/no-get-with-default.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const types = require('../utils/types');
+
+function makeErrorMessageForGetWithDefault(property) {
+  return `Use \`get\` and \`||\` instead of \`getWithDefault\` for property \`${property}\``;
+}
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Use the || operator instead of getWithDefault for more expected behaviors',
```

Comment:
`for more expected behaviors` isn't really helpful so I would leave that part out.